### PR TITLE
MHV-Health Conditions and Care Summaries contribution.

### PR DESCRIPTION
**Pull request by Wyzardsleeves**
## Summary

- Edited mock data and made some HTML changes for formatting.
- Not a bug.
- JSX and JSON work. 
- MHV Medical Records.

## Testing done

-  Updated Unit test for JSX.

## What areas of the site does it impact?
MHV Medical Records

### CODE REVIEW

**Review by mdewey — DISMISSED** [review]

File: src/applications/mhv-medical-records/reducers/careSummariesAndNotes.js
Comment (mattwrightva) [inline src/applications/mhv-medical-records/reducers/careSummariesAndNotes.js:1]: Why did you change this to a string?

**Review by mattwrightva — APPROVED** [review]
Looks good!

**Review by mdewey — APPROVED** [review]
